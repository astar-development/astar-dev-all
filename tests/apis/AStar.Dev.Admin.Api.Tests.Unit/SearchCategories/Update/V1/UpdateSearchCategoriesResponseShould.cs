using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Admin.Api.SearchCategories.Update.V1;

[TestSubject(typeof(UpdateSearchCategoryResponse))]
public sealed class UpdateSearchCategoriesResponseShould
{
    [Fact]
    public void ContainTheExpectedValues()
    {
        var sut = new UpdateSearchCategoryResponse(new()
                                                   {
                                                       Name                = "Name",
                                                       UpdatedBy           = "UpdatedBy",
                                                       UpdatedOn           = new(2001, 1, 1),
                                                       TotalPages          = 1,
                                                       Id                  = 2,
                                                       LastPageVisited     = 3,
                                                       SearchCategoryId    = Guid.Empty,
                                                       LastKnownImageCount = 4,
                                                       Order               = 5,
                                                   });

        sut.ToJson().ShouldMatchApproved();
    }
}
