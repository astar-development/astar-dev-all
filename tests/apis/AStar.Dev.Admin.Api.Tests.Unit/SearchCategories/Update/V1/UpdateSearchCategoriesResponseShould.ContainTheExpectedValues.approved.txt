{
  "updatedOn": "2001-01-01T00:00:00",
  "name": "Name",
  "updatedBy": "UpdatedBy",
  "order": 5,
  "totalPages": 1,
  "lastPageVisited": 3,
  "lastKnownImageCount": 4,
  "searchCategoryId": "00000000-0000-0000-0000-000000000000"
}